media-tv/plex-media-server/plex-media-server-1.16.4.ebuild

```diff
@@ -0,0 +1,157 @@
+	keepdir "${LOGGING_DIR}"
+
+	# Create default library folder with correct permissions
+	local DEFAULT_LIBRARY_DIR="/var/lib/${_APPNAME}"
+	dodir "${DEFAULT_LIBRARY_DIR}"
+	chown "${_USERNAME}":"${_USERNAME}" "${ED}/${DEFAULT_LIBRARY_DIR}" || die
+	keepdir "${DEFAULT_LIBRARY_DIR}"
+
+	# Install the OpenRC init/conf files
+	doinitd "${FILESDIR}/init.d/${PN}"
+	doconfd "${FILESDIR}/conf.d/${PN}"
+
+	# Disabling due to Bug 644694
+	#_handle_multilib
+
+	# Mask Plex libraries so that revdep-rebuild doesn't try to rebuild them.
+	# Plex has its own precompiled libraries.
+	_mask_plex_libraries_revdep
+
+	# Install systemd service file
+	local INIT_NAME="${PN}.service"
+	local INIT="${FILESDIR}/systemd/${INIT_NAME}"
+	systemd_newunit "${INIT}" "${INIT_NAME}"
+
+	_remove_execstack_markings
+	_add_pax_markings
+
+	einfo "Configuring virtualenv"
+	virtualenv -v --no-pip --no-setuptools --no-wheel "${ED}"/usr/lib/plexmediaserver/Resources/Python || die
+	pushd "${ED}"/usr/lib/plexmediaserver/Resources/Python &>/dev/null || die
+	find . -type f -exec sed -i -e "s#${D}##g" {} + || die
+	popd &>/dev/null || die
+}
+
+pkg_postinst() {
+	einfo ""
+	elog "Plex Media Server is now installed. Please check the configuration file in /etc/${_SHORTNAME}/${_APPNAME} to verify the default settings."
+	elog "To start the Plex Server, run 'rc-config start plex-media-server', you will then be able to access your library at http://<ip>:32400/web/"
+}
+
+# Disabling the follow function due to Bug 644694.
+# We shouldn't register plex libraries in global
+# library path since this will cause other packages
+# on the system to break.
+
+# Finds out where the library directory is for this system
+# and handles ldflags as to not break library dependencies
+# during rebuilds.
+_handle_multilib() {
+	# Prevent revdep-rebuild, @preserved-rebuild breakage
+	cat > "${T}"/66plex <<-EOF || die
+		LDPATH="${EPREFIX}/usr/$(get_libdir)/plexmediaserver"
+	EOF
+
+	doenvd "${T}"/66plex
+}
+
+# Adds the precompiled plex libraries to the revdep-rebuild's mask list
+# so it doesn't try to rebuild libraries that can't be rebuilt.
+_mask_plex_libraries_revdep() {
+	dodir /etc/revdep-rebuild/
+
+	# Bug: 659702. The upstream plex binary installs its precompiled package to /usr/lib.
+	# Due to profile 17.1 splitting /usr/lib and /usr/lib64, we can no longer rely
+	# on the implicit symlink automatically satisfying our revdep requirement when we use $(get_libdir).
+	# Thus we will match upstream's directory automatically. If upstream switches their location,
+	# then so should we.
+	echo "SEARCH_DIRS_MASK=\"${EPREFIX}/usr/lib/plexmediaserver\"" > "${ED}"/etc/revdep-rebuild/80plexmediaserver
+}
+
+# Remove execstack flags from some libraries/executables
+# so that it works in hardened setups.
+_remove_execstack_markings() {
+	for f in "${EXECSTACKED_BINS[@]}"; do
+		# Unquoting 'f' so that expansion works.
+		fix-gnustack -f ${f} > /dev/null
```

Comment:
`|| die`

Reply:
Removed as the file that was meant to be marked doesn't exist in this version of Plex.